src/components/pages/GamePlay/GameDrawStep.js

```diff
@@ -134,13 +135,13 @@ export default function GameDrawStep(props) {
                 onClick={handleSubmit}
                 loading={isUploading}
               >
-                {isUploading ? 'Uploading...' : 'Save'}
+                {isUploading ? 'Uploading...' : 'Submit'}
```

Comment:
I like this change a lot, especially when it's directed at children who will hear "submit" in school in reference to turning in their work, so I believe it will resonate more with them than "save"

Reply:
Beautiful comments. Thanks a lot for those.